radicle-cli/src/commands/remote/rm.rs

```diff
@@ -0,0 +1,11 @@
+use crate::git;
+use crate::terminal as term;
+
+pub fn run(repository: &git::Repository, alias: &str) -> anyhow::Result<()> {
+    if !git::rad_has_remote(repository, alias)? {
+        anyhow::bail!("remote with alias {alias} not found!");
+    }
```

Comment:
Maybe use rad_remotes() + Rust's [any method](https://doc.rust-lang.org/1.2.0/std/iter/trait.Iterator.html#method.any)? i.e. should `rad remote rm` work on non-radicle remotes?

Reply:
Agreed :+1: